src/lib/ecl

```diff
@@ -1 +1 @@
-Subproject commit 74a77b45b2b51cb9a2f2a05e0b356d87cac0d9a8
```

Comment:
@bkueng This looks like a bad merge which shouldn't have been committed. Can you update this to 1b394460a39b6f69b913383e0b9e66527f617650 (HEAD) instead?